plugins/interactive.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: interactive               
```

Comment:
I'd recommend a more specific name, [per the naming guide](https://github.com/GoogleContainerTools/krew/tree/master/docs/NAMING_GUIDE.md). Perhaps something like `interactive-prompt`, `prompt-shell`, `interactive-shell`, or `it-shell`